stimela/cargo/cab/msutils/parameters.json

```diff
@@ -47,7 +47,8 @@
             "default": null, 
             "name": "outfile", 
             "io": "output", 
-            "dtype": "file"
+            "dtype": "file",
+            "check_io": false,
```

Comment:
It's strange that this needs to be done for an output product. Unless there is a bug in the validation in the cab module

Reply:
I guess it is a bug in the validation. The problem is here: https://github.com/ratt-ru/Stimela/blob/master/stimela/cargo/cab/__init__.py#L349

If I override the location of the output file using ``:msfile``, it starts validating it as if it's an input parameter, based on the new location.